cdap-docs/included-applications/source/etl/plugins/transforms/validator.rst

```diff
@@ -70,6 +70,31 @@ using ``"validators": "core"``) and references a function using its Javascript n
       {
         "name": "Validator",
         "properties": {
+          "lookup": "{
+            \"blacklist\":{
+              \"type\":\"DATASET\"
+            }
+          }"
+          "validationScript": "function isValid(input, context) {
+                                  if (context.getLookup('blacklist').lookup(input.body) !== null)
```

Comment:
Where do we document which functions are supported by the object returned by ``getLookup()``?